config-connector/tests/testcases/environments.template

```diff
@@ -11,5 +11,10 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+# RANDOM_ID should not be defined in the environments file.
+BILLING_ACCOUNT_ID: ${BILLING_ACCOUNT_ID?}
 DEFAULT_PROJECT_ID: ${DEFAULT_PROJECT_ID?}
 IAM_MEMBER: ${IAM_MEMBER?}
+ORG_ID: ${ORG_ID?}
+SIMPLE_PROJECT_ID: ${SIMPLE_PROJECT_ID?}
```

Comment:
Now we have DEFAULT_PROJECT_ID and SIMPLE_PROJECT_ID. Why not just have a PROJECT_ID and use it for both? Worried that as we are adding more test cases a number of different SOMETHING_PROJECT_ID will grow.

Reply:
SOMETHING_PROJECT_ID env vars (except for DEFAULT_PROJECT_ID, which is for an existing project) are used as the name prefix for new projects to create. Because project ID needs to be a globally unique value, they should be different for each test run for each solution. Currently there are only 4 solutions creating 5 projects in total so there will only be five SOMETHING_PROJECT_ID env vars. Will look into the improvements after finishing all the test cases.